dev/ci/ci-basic-overlay.sh

```diff
@@ -22,11 +22,11 @@
 : "${oddorder_CI_ARCHIVEURL:=${oddorder_CI_GITURL}/archive}"
 
 ########################################################################
-# UniMath
+# unimath
 ########################################################################
-: "${UniMath_CI_REF:=master}"
-: "${UniMath_CI_GITURL:=https://github.com/UniMath/UniMath}"
-: "${UniMath_CI_ARCHIVEURL:=${UniMath_CI_GITURL}/archive}"
+: "${unimath_CI_REF:=master}"
+: "${unimath_CI_GITURL:=https://github.com/unimath/unimath}"
```

Comment:
Umm, I'd rather respect the upstream capitalization. Assuming things are case-insensitive is not cool.